content/hi/mutual-transport-layer-security.md

```diff
@@ -0,0 +1,25 @@
+---
+title: एमटीएलएस (Mutual Transport Layer Security)
+status: Completed
+category: अवधारणा
+tags: ["security", "", ""]
+---
+
+## यह क्या है
+
+म्युचुअल टीएलएस (एमटीएलएस) एक तकनीक है जिसका उपयोग दो [सेवाओं](/service/) के बीच भेजे गए संदेशों को प्रमाणित और एन्क्रिप्ट करने के लिए किया जाता है।
+म्यूचुअल टीएलएस मानकी [ट्रांसपोर्ट लेयर सिक्योरिटी](/transport-layer-security/) (टीएलएस) प्रोटोकॉल है लेकिन,
+केवल एक कनेक्शन की पहचान को मान्य करने के बजाय, दोनों पक्षों को मान्य किया जाता है।
```

Comment:
- remove extra space in the paragraph, sentences should be in continuation.
- what is `मानकी` ?